content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -104,18 +98,20 @@ istio-pilot-58c65f74bc-2f5xn             1/1       Running   0          1m
 
 * For option 2, uninstall using Helm:
 
+> Uninstalling this chart does not delete Istio's registered CRDs. Istio, by design, expects
+> CRDs to leak into the Kubernetes environment. As CRDs contain all runtime configuration
+> data for Istio configuration the Istio designers consider better to explicitly delete this
```

Comment:
This sentence is klunky.  Recommend breaking it up with periods.  `CRDs contain all runtime configuration for Istio configuration.  The Istio designers consider it better for operators to explicitly delete the runtime configuration rather than unexpectedly lose it.

Reply:
I agree, what about:
```suggestion
CRDs contain all the runtime configuration data needed to configure Istio. Because of this, we consider it better for operators to explicitly delete the runtime configuration data rather than unexpectedly lose it.
```

Reply:
Done